src/routes/events/events.router.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const { check } = require('express-validator');
+
+const {
+  listEvents,
+  postNewEvent,
+  updateEvent,
+  deleteEvent
+} = require('./events.controller');
```

Comment:
How could you change this so that the function names are just the action to be taken (e.g. list, post, etc) *without* losing the context that they're operating on an Event?

Reply:
Hmm, they are already in the events controller, is that enough context? Like just

```
const {
 list,
 post,
 update,
 delete
} = require ('./events.controller');

Reply:
Close! The problem with that approach is that when you hook up a route to the method, you wouldn't have that Event context immediately evident.  That would also break down as soon as you have more than one resource.

However if you did something like this:

```
cost events = require('./events.controller');
```

Then you could hook them up like

```
events.list
```